fmatch/matcher.py

```diff
@@ -0,0 +1,126 @@
+from elasticsearch7 import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+import pandas as pd
+
```

Comment:
lets add a logging lib to help the user... some of the commented prints could be `logger.debug`  messages and we default the logger to info. thoughts?

Reply:
Yes adding logger should be helpful if we move towards CLI.